docs/en/query_language/select.md

```diff
@@ -80,6 +80,29 @@ A sample with a relative coefficient is "consistent": if we look at all possible
 
 For example, a sample of user IDs takes rows with the same subset of all the possible user IDs from different tables. This allows using the sample in subqueries in the IN clause, as well as for manually correlating results of different queries with samples.
 
+**SAMPLE OFFSET**
+
+You can specify the `SAMPLE k OFFSET n` clause, where `k` and `n` are decimal numbers from 0 to 1. Examples are shown below.
```

Comment:
Decimal is a separate data type that doesn't have anything to do with these numbers, I'd just skip this word.